### PR TITLE
[cisco_meraki] Add feature to remove original event

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Add the ability to remove `event.original` when `preserve_original_event` is not in `tags`.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Comment by LaZyDK** [comment]
@P1llus test :)

File: packages/cisco_meraki/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.1"
```
Comment (efd6) [inline packages/cisco_meraki/changelog.yml:2]: s/1.1.1/1.2.0/ as it's an enhancement.

**Pull request merged**